src/styles/index.scss

```diff
@@ -1,3 +1,92 @@
-body {
+$stopwatch-width: 80vmin;
+$stopwatch-border: 1vmin dotted #000;
+$minutes-length: 20vmin;
+$minutes-width: 3vmin;
+$minutes-color: #0700ff;
+$seconds-length: 38vmin;
+$seconds-width: 1.5vmin;
+$seconds-color: #2c8000;
+$center-width: 5vmin;
+$center-color: #f6a603;
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  border: $stopwatch-border;
+  width: $stopwatch-width;
+  height: $stopwatch-width;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
```

Comment:
you have a lot of doublicated position properties, please use scss [placeholders](https://sass-lang.com/documentation/style-rules/placeholder-selectors/)